tests/unit/test__helpers.py

```diff
@@ -203,6 +203,38 @@ def test_success_with_retry(self, randint_mock, sleep_mock):
         sleep_mock.assert_any_call(2.625)
         sleep_mock.assert_any_call(4.375)
 
+    @mock.patch(u"time.sleep")
+    @mock.patch(u"random.randint")
+    def test_success_with_retry_custom_delay(self, randint_mock, sleep_mock):
+        randint_mock.side_effect = [125, 625, 375]
```

Comment:
Mapping these values onto the expected `sleep` calls below took a fair bit of mental gymnastics.  Maybe we could make it more legible, e.g.:

```python
initial_delay = 3.0  # sec
multiplier = 4
random_jitter = randint_mock.side_effect = [125, 625, 375]  # ms
expected_sleeps = [
    initial_delay * multiplier * index + jitter
    for index, jitter in enumerate(random_jitter)
]
```

And then below:

```python
assert sleep_mock.assert_has_calls([
    mock.call(expected_sleep) for expected_sleep in expected_sleeps
])
```

Reply:
In the interest of being uniform across all similar tests, I'll take a shot at clarifying this with comments first.

Reply:
PTAL; do these comments clarify sufficiently?

Reply:
The comments do clarify the intent.  I'm not sure why this module prefers the count + `assert_any_call` bit is preferred over the stronger `assert_has_calls` (which tests order by default), but we can let that go here.